CMakeLists.txt

```diff
@@ -14,14 +14,14 @@ if (WIN32)
 	set(KIMAGEANNOTATOR_LANG_INSTALL_DIR "translations")
 
 	set(CPACK_GENERATOR WIX)
-	set(CPACK_PACKAGE_NAME "ksnip")
+	set(CPACK_PACKAGE_NAME "ksnip Screenshot Tool")
 	set(CPACK_PACKAGE_VENDOR "ksnip")
+	set(CPACK_WIX_UPGRADE_GUID "4c7ed545-c0dd-4d45-bf69-c29c7998f668")
 	set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Ksnip is a Qt-based cross-platform screenshot tool that provides many annotation features for your screenshots.")
-	set(CPACK_PACKAGE_VERSION "0.1.0")
-	set(CPACK_PACKAGE_VERSION_MAJOR "0")
-	set(CPACK_PACKAGE_VERSION_MINOR "1")
-	set(CPACK_PACKAGE_VERSION_PATCH "0")
+	set(CPACK_PACKAGE_VERSION "${PROJECT_VERSION}")
 	set(CPACK_PACKAGE_INSTALL_DIRECTORY "ksnip")
+	set(CPACK_RESOURCE_FILE_LICENSE "../LICENSE.txt")
```

Comment:
Does the LICENSE file need to be renamed to .txt? It is a txt file, no question about that but we have now a mix of txt and md files.

Reply:
it needs an extension at least. [.md would work as well](https://cmake.org/cmake/help/git-stage/cpack_gen/nuget.html#variable:CPACK_NUGET_%3CcompName%3E_PACKAGE_LICENSE_FILE_NAME).. but without extension cpack does not know what to do with it.